### PR TITLE
Fix verifyEnumerable helper to account for properties with Symbol values

### PR DESCRIPTION
**Pull request by leobalter**

### CODE REVIEW

File: test/harness/propertyhelper-verifyenumerable-not-enumerable.js [hunk 1]
```diff
@@ -6,26 +6,52 @@ description: >
...
+  }
+}
+
+if (threwStringKey === false) {
+  throw new Test262Error(
+    'Expected a Test262Error, but no error was thrown for string key.'
+  );
+}
+
+if (threwStringKey === false) {
```
Comment (jugglinmike) [inline test/harness/propertyhelper-verifyenumerable-not-enumerable.js]: This should be `threwSymbolKey`

File: test/harness/propertyhelper-verifynotenumerable-enumerable.js [hunk 2]
```diff
@@ -26,6 +32,26 @@ try {
...
+  }
+}
+
+if (threwStringKey === false) {
+  throw new Test262Error(
+    'Expected a Test262Error, but no error was thrown for string key.'
+  );
 }
+
+if (threwStringKey === false) {
```
Comment (jugglinmike) [inline test/harness/propertyhelper-verifynotenumerable-enumerable.js]: Same here--this should be `threwSymbolKey`

**Comment by jugglinmike** [comment]
Generally, I think it would be better to make these new assertions from separate files. Doing so will improve readability (since independent assertions will not be interleaved) but more importantly, it will allow hosts that do not implement Symbol to verify the correctness of these helper functions.

**Comment by leobalter** [comment]
Thanks for the feedback, I applied the suggestion on the rebased commit.

**Comment by leobalter** [comment]
can we merge this?

**Pull request merged**